### PR TITLE
fees panel focus improvement + fix e2e

### PR DESCRIPTION
**Pull request by estebanmino**
Fixes RNBW-####

## What changed (plus any additional context for devs)

- fix e2e
- setting `custom` in the context menu wasn't opening the custom gas sheet correctly

## Dev checklist for QA: what to test

choose custom from the context menu in gas speed button

## Final checklist
[ ] Assigned individual reviewers?
[ ] Added labels?

### CODE REVIEW

**Review by brunobar79 — APPROVED** [review]
🙏 🙏 🙏

File: src/components/gas/GasSpeedButton.js [hunk 1]
```diff
@@ -260,8 +260,8 @@ const GasSpeedButton = ({
       }
     },
     [
-      openCustomGasSheet,
       gasFeeParamsBySpeed,
+      setShouldOpenCustomGasSheet,
```
Comment (jinchung) [inline src/components/gas/GasSpeedButton.js:264]: oh I missed this:
`setShouldOpenCustomGasSheet` can be removed from all of the dependency lists in this file

Comment (jinchung) [reply src/components/gas/GasSpeedButton.js:264]: there are more places it is added the the dependencies than just here

**Review by jinchung — APPROVED** [review]
🌮

**Pull request merged**